debug_toolbar/templates/debug_toolbar/base.html

```diff
@@ -16,7 +16,7 @@
      data-sidebar-url="{{ history_url }}"
      {% endif %}
      data-default-show="{% if toolbar.config.SHOW_COLLAPSED %}false{% else %}true{% endif %}"
-     {{ toolbar.config.ROOT_TAG_EXTRA_ATTRS|safe }}>
+     {{ toolbar.config.ROOT_TAG_EXTRA_ATTRS|safe }}  data-update-on-fetch="{{ toolbar.config.UPDATE_ON_FETCH|safe }}">
```

Comment:
```suggestion
     {{ toolbar.config.ROOT_TAG_EXTRA_ATTRS|safe }}  data-update-on-fetch="{{ toolbar.config.UPDATE_ON_FETCH }}">
```

I'm not 100% sure but I don't think `|safe` is necessary.

Also, you could use `|yesno:'true,false'` or do it as `data-default-show` so that the value is lowercased already, but I'm just suggesting that, not asking for anything.

Reply:
I removed safe. But I didn't user yesno because the result of this will be a string and we will need do to the same thing in js for checking it lowercased or not.